src/awkward/forms/bitmaskedform.py

```diff
@@ -1,12 +1,44 @@
 # BSD 3-Clause License; see https://github.com/scikit-hep/awkward-1.0/blob/main/LICENSE
 
+import copy
+
 import awkward as ak
+from awkward._util import unset
 from awkward.forms.form import Form, _parameters_equal
 
 
 class BitMaskedForm(Form):
     is_option = True
 
+    def copy(
+        self,
+        mask=unset,
+        content=unset,
+        valid_when=unset,
+        lsb_order=unset,
+        *,
+        parameters=unset,
+        form_key=unset,
+    ):
+        return BitMaskedForm(
+            self._mask if mask is unset else mask,
+            self._content if content is unset else content,
+            self._valid_when if valid_when is unset else valid_when,
+            self._lsb_order if lsb_order is unset else lsb_order,
+            parameters=self._parameters if parameters is unset else parameters,
+            form_key=self._form_key if form_key is form_key else form_key,
```

Comment:
```suggestion
            form_key=self._form_key if form_key is unset else form_key,
```

Not sure if you're taking review at draft stage, but I noticed this and flagged it here :)

Reply:
Got it, thanks!